.github/workflows/run_codecov_on_pull_request.yml

```diff
@@ -23,29 +22,29 @@ jobs:
           node-version: "18"
 # Need to run "npm run build" command for bundle analysis: https://docs.codecov.com/docs/webpack-quick-start
       - name: Build React and run tests
-        env:
-            CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
         run: |
           CI=false ./build.sh
           npm test -- --coverage --reporters=jest-junit || true
-        working-directory: ./empower/react
-      - uses: codecov/codecov-action@v3
+        working-directory: ./react
+      - uses: codecov/codecov-action@v5
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
           flags: frontend
+          verbose: true
 
       - name: Run Python Tests
         run: |
           pip install -U pytest
           pip install pytz
           pip install pytest-codecov
           pytest --cov=. --cov-report=xml --junitxml=junit.xml -o junit_family=legacy
-        working-directory: ./empower/flask/src
+        working-directory: ./flask/src
```

Comment:
@thomasrockhu-codecov just curious, is this somehow necessary for it to work (e.g. ignores `working-directory` when looking for codecov.yml and always looks in cwd) or just aesthetic?

Reply:
this is necessary because the `checkout` step doesn't shove the repository into `empower`. This has nothing to do with the yml

Reply:
i.e. if `working-direcotry` is not changed here, the commands above will fail because the directory structure has changed with the removal of 

`path: empower`

Reply:
@thomasrockhu-codecov yes, I saw that :-) 
was the removal of `path: empower` necessary or aesthetic?

Reply:
@realkosty it is probably possible to get it work with `path: empower`, but I don't see why it was necessary in the workflow to begin with.